Source/AliveLib/Scrab.cpp

```diff
@@ -627,8 +628,483 @@ __int16 Scrab::AI_Patrol_0_4AA630()
+        {
+            field_120_obj_id = -1;
+            field_108_next_motion = eScrabMotions::M_HowlBegin_26_4A9DA0;
+            Scrab::ToPatrol_4AA600();
+            result = 9;
+        }
+        else
+        {
+            if (Scrab::CanSeeAbe_4A51A0(v6))
+            {
+                field_14C = sGnFrame_5C1B84 + field_148_attack_duration;
+            }
+            if (Event_Is_Event_In_Range_422C30(kEventAbeOhm, field_B8_xpos, field_BC_ypos, -1) && field_11C_sub_state != 26)
+            {
+                field_108_next_motion = eScrabMotions::M_HowlBegin_26_4A9DA0;
+                result = 4;
+            }
+            else
+            {
+                pObj = sObjectIds_5C1B70.Find_449CF0(field_110_id);
+                LiftPoint* pLiftPoint = nullptr;
+                pLiftPoint = static_cast<LiftPoint*>(pObj);
+                if (!pObj || pObj->field_4_typeId != Types::eLiftPoint_78 || pLiftPoint->vOnAnyFloor_461920() || field_11C_sub_state == 4)
+                {
+                    if (gMap_5C3030.GetDirection_4811A0(
+                        field_C2_lvl_number, 
+                        field_C0_path_number, 
+                        field_B8_xpos, 
+                        field_BC_ypos) >= CameraPos::eCamCurrent_0)
+                    {
+                        MusicController::sub_47FD60(8, this, 0, 0);
+                    }
+                    switch (field_11C_sub_state)
+                    {
+                    case 0:
+                        result = 1;
+                        field_150 = sGnFrame_5C1B84 + field_128_attack_delay;
+                        return result;
+                    case 1:
+                        if (!field_106_current_motion)
+                        {
+                            field_194 = static_cast<short>(Scrab::LastSpeak_4A56F0());
+                            if (field_1A2 < field_1A0 
+                                && Scrab::LastSpeak_4A56F0() == GameSpeakEvents::eUnknown_53 
+                                || Scrab::LastSpeak_4A56F0() == GameSpeakEvents::eUnknown_54)
+                            {
+                                return 15;
+                            }
+                        }
+                        if (!vIsFacingMe_4254A0(v6))
+                        {
+                            if (gMap_5C3030.Is_Point_In_Current_Camera_4810D0(field_C2_lvl_number, field_C0_path_number, field_B8_xpos, field_BC_ypos, 0))
+                            {
+                                field_108_next_motion = eScrabMotions::M_Turn_3_4A91A0;
+                            }
+                            else
+                            {
+                                field_106_current_motion = eScrabMotions::M_Turn_3_4A91A0;
+                                field_108_next_motion = -1;
+                                BaseAliveGameObject::MapFollowMe_408D10(1);
+                            }
+                            result = 3;

```

Comment:
result can be removed from this function and just return the value directly with `return 3`;